tmh_registry/registry/admin.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib import admin
+
+from tmh_registry.registry.models import (
+    Hospital,
+    Patient,
+    PatientHospitalMapping,
+)
+from tmh_registry.users.models import MedicalPersonnel
+
+
+@admin.register(Hospital)
+class HospitalAdmin(admin.ModelAdmin):
+    model = Hospital
+
+
+@admin.register(Patient)
+class PatientAdmin(admin.ModelAdmin):
+    model = Patient
+
+
+@admin.register(MedicalPersonnel)
```

Comment:
You need to remove the MedicalPersonnel admin registration - it is already registered in the `users` app.